src/createServer.js

```diff
@@ -1,3 +1,33 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require(('./convertToCase/convertToCase'));
+const { validaton } = require('./validation');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+    const textToConvert = normalizedURL.pathname.slice(1);
+    const toCase = normalizedURL.searchParams.get('toCase');
+    const errors = validaton(textToConvert, toCase);
+
+    if (errors.errors.length) {
+      req.statusCode = 400;
+      req.statusMessage = 'Bad request';
+      res.end(JSON.stringify(errors));
+    } else {
+      res.statusMessage = 'OK';
+
+      const convertedText = convertToCase(textToConvert, toCase);
+
+      res.end(JSON.stringify({
+        ...convertedText,
+        targetCase: toCase,
+        originalText: textToConvert,
+      }));
+    }
+  });
+
+  return server;
+}
+module.exports = { createServer };
```

Comment:
Indentation

Reply:
Already fix this issue, but i will push new commit after mentor rewiev)